setup.py

```diff
@@ -77,5 +77,6 @@
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
This will add 3.7 to the `pypi supported versions` badge in README (probably in next release)